spec/defines/foreman_plugin_spec.rb

```diff
@@ -29,10 +30,7 @@
     } end
 
     it 'should install the correct package with notify' do
```

Comment:
Right, so now the notify is gone that part of the description no longer makes sense.